op-conductor/conductor/flashblocks_handler.go

```diff
@@ -0,0 +1,209 @@
+package conductor
```

Comment:
1. recommend to put this file under `rpc` folder or create a new `ws` folder
2. rename the file to `flashblocks_proxy.go`